Fika.Core/FikaPlugin.cs

```diff
@@ -263,6 +268,9 @@ private void SetupConfig()
 
             AcceptedTOS = Config.Bind("Hidden", "Accepted TOS", false, new ConfigDescription("Has accepted TOS", tags: new ConfigurationManagerAttributes() { Browsable = false }));
 
+            // Advanced
+            OfficialVersion = Config.Bind("Advanced", "Official Version", false, new ConfigDescription("Show official version instead of fika version.", tags: new ConfigurationManagerAttributes() { IsAdvanced = true }));
```

Comment:
```suggestion
            OfficialVersion = Config.Bind("Advanced", "Official Version", false, new ConfigDescription("Show official version instead of Fika version.", tags: new ConfigurationManagerAttributes() { IsAdvanced = true }));
```

Capitalize the name here.